be/src/olap/rowset/segment_v2/inverted_index_reader.cpp

```diff
@@ -47,29 +48,38 @@ bool InvertedIndexReader::indexExists(io::Path& index_file_path) {
     return exists;
 }
 
-std::vector<std::string> FullTextIndexReader::get_analyse_result(
-        const std::wstring& field_name, const std::wstring& value,
-        InvertedIndexQueryType query_type, InvertedIndexParserType analyser_type) {
-    std::vector<std::string> analyse_result;
+std::vector<std::wstring> FullTextIndexReader::get_analyse_result(
+        const std::wstring& field_name, const std::string& value, InvertedIndexQueryType query_type,
+        InvertedIndexParserType analyser_type) {
+    std::vector<std::wstring> analyse_result;
     std::shared_ptr<lucene::analysis::Analyzer> analyzer;
+    std::unique_ptr<lucene::util::Reader> reader;
     if (analyser_type == InvertedIndexParserType::PARSER_STANDARD) {
         analyzer = std::make_shared<lucene::analysis::standard::StandardAnalyzer>();
+        reader.reset(
+                (new lucene::util::StringReader(std::wstring(value.begin(), value.end()).c_str())));
+    } else if (analyser_type == InvertedIndexParserType::PARSER_CHINESE) {
+        analyzer = std::make_shared<lucene::analysis::LanguageBasedAnalyzer>(L"chinese", false);
```

Comment:
need to set dict path for chinese analyzer here

Reply:
fixed